presto-main/src/main/java/io/prestosql/cost/CachingCostProvider.java

```diff
@@ -40,7 +40,7 @@
     private final Session session;
     private final TypeProvider types;
 
-    private final Map<PlanNode, PlanNodeCostEstimate> cache = new IdentityHashMap<>();
+    private final Map<PlanNode, PlanCostEstimate> cache = new IdentityHashMap<>();
```

Comment:
rename it to `CumulativePlanCostEstimate`

Reply:
i prefer the current name more